src/transformers/tokenization_dpr.py

```diff
@@ -203,15 +203,37 @@ class CustomDPRReaderTokenizerMixin:
     def __call__(
         self,
         questions,
-        titles,
-        texts,
+        titles: Optional[str] = None,
+        texts: Optional[str] = None,
         padding: Union[bool, str] = True,
         truncation: Union[bool, str] = True,
-        max_length: Optional[int] = 512,
+        max_length: Optional[int] = None,
         return_tensors: Optional[Union[str, TensorType]] = None,
         return_attention_mask: Optional[bool] = None,
         **kwargs
     ) -> BatchEncoding:
+        if titles is None and texts is None:
```

Comment:
I had to handle the cases of having only one or two input strings for the tests.